blogs/jhylin.github.io.json

```diff
@@ -0,0 +1,19 @@
+{
+  "title": "Data in life",
+  "type": "blog",
+  "url": "https://jhylin.github.io/Data_in_life_blog/",
+  "photo_url": "https://github.com/jhylin/Data_in_life_blog/blob/main/profile_avatar.jpg",
+  "description": "A blog using data science on pharmaceutical and healthcare data",
+  "language": "en", 
+  "rss_feed": "https://jhylin.github.io/Data_in_life_blog/index-r.xml" 
```

Comment:
Here should be a comma after your feed entry ☺️

Reply:
Thank you @cosimameyer, I've just added the comma.

Reply:
Good point, thanks so much! ☺️